docs/team/alvin.md

```diff
@@ -0,0 +1,40 @@
+---
+layout: page
+title: Alvin's Project Portfolio Page
+---
+
+## Project: ATAS
+
+**ATAS** is a desktop application designed to be a handy in-class companion for CS1010S Teaching Assistants. 
+It aims to **elevate users' efficiency** and make their experience of handling administrative tasks as **seamless** as possible. 
+The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Added the ability to enter a session.
+  * What it does: allows the user to enter a session to view the presence and participation status of the students.
+  * Justification: This feature limits the user to toggle the presence and participation status of the students only when he or she is in a session.
+  * Highlights: The implementation was quite tricky as it required changes to existing commands and the usage had to be limited to a particular tab.
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=CodingCookieRookie&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+
+* **Enhancements to existing features**:
+  * Updated the status bar footer to show current session details(Pull requests [\#167](https://github.com/AY2021S1-CS2103T-W16-4/tp/pull/167))
```

Comment:
Perhaps "Pull request" instead of "Pull requests"? Also maybe an additional whitespace before the bracket may make it look nicer? I see this issue elsewhere too.

Reply:
Noted with thanks :D